src/components/form-checkbox.tsx

```diff
@@ -0,0 +1,24 @@
+// outsource dependencies
+import { useField } from 'formik';
+import React, { memo } from 'react';
+import { Checkbox, CheckboxProps, FormControlLabel, FormHelperText } from '@mui/material';
+
+interface FCheckboxProps extends CheckboxProps {
+  name: string
+  label?: string
+}
+
+const FCheckbox: React.FC<FCheckboxProps> = ({ label, name, ...attr }) => {
+  const [field, meta] = useField({ name });
+  return <>
+    <FormControlLabel control={<Checkbox {...attr} {...field} onChange={field.onChange}/>} label={label}/>
+    { meta.touched && meta.error && <FormHelperText
```

Comment:
```suggestion
    { meta.invalid && <FormHelperText
```
  const invalid = (skipTouch || meta.touched) && !!meta.error;
  const valid = (skipTouch || meta.touched) && !meta.error;